src/generators/utils.js

```diff
@@ -38,12 +38,11 @@ module.exports = {
   requireHerbarium: (command, appPath) => {
     const herbariumPath = `${appPath}/src/domain/herbarium.js`
     let herbarium
-    if (command === "update" || command === "spec") {
+    if (command === "update" || command === "spec" || command === "shell") {
```

Comment:
I think you can change this line to:
`if (command !== "new")`... but need to test it